packages/react-components/react-carousel/library/src/components/useEmblaCarousel.ts

```diff
@@ -114,7 +122,7 @@ export function useEmblaCarousel(
     }
 
     return plugins;
-  }, [motion, onDragEvent, watchDrag]);
+  }, [motion, onDragEvent, watchDrag, autoplayInterval]);
```

Comment:
If changes to `motion` will be done, it would be safer to do descructing outside, so we don't have an object in dependencies i.e.

```
[motionKind, motionDuration, onDragEvent, ...]
```